### PR TITLE
added ClearSight, Jamon Holmgren, & Ryan Linton Gems

### PR DESCRIPTION
**Pull request by kevinvangelder**
@jamonholmgren @ryanlntn @silasjmatson

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -138,9 +138,20 @@
           "url": "https://github.com/clearsightstudio/ProMotion",
           "description": "A full featured RubyMotion framework that makes iPhone development less like Objective-C and more like Ruby, designed to get up and running fast.",
           "tags": [
+            "ProMotion",
             "Storyboard",
```
Comment (jamonholmgren) [inline data.json]: Remove Storyboard.

**Comment by jamonholmgren** [comment]
:+1:

File: data.json [hunk 2]
```diff
@@ -1434,6 +1458,127 @@
...
+            "iOS"
+          ]
+        },
+        {
+          "name": "ProMotion-menu",
+          "gem_name": "ProMotion-menu",
+          "url": "https://github.com/clearsightstudio/ProMotion-menu",
+          "description": "RubyMotion gem allowing you to easily setup a facebook or Path style hidden slide menu easily with the ProMotion gem.",
+          "tags": [
+            "NSMenu",
```
Comment (ryanlntn) [inline data.json]: Remove `NSMenu`. That's AppKit.

**Comment by ryanlntn** [comment]
:+1:

**Pull request merged**